src/tools/PlaywrightSharp.Tooling/DriverDownloader.cs

```diff
@@ -155,10 +156,12 @@ private async Task<bool> ExecuteAsync()
 
                 var tasks = new List<Task>();
 
+                /*
```

Comment:
I had to comment this out for now, because the release isn't actually created. We might have more of these in the future, so I'll think more about how to sync this (if at all).

Reply:
How about default to master on `UpdateBrowserVersionsAsync`, if we don't find a matching tag?

Reply:
Good point, I've added a fallback to `.0` and exception catching with a warning printout, rather than failing the whole process.